src/test/java/com/internship/frejaeidjmeterplugin/jmeter/frejaRequests/SignServiceTest.java

```diff
@@ -0,0 +1,99 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package com.internship.frejaeidjmeterplugin.jmeter.frejaRequests;
+
+import com.verisec.frejaeid.client.beans.sign.get.SignResult;
+import com.verisec.frejaeid.client.enums.MinRegistrationLevel;
+import com.verisec.frejaeid.client.exceptions.FrejaEidClientInternalException;
+import com.verisec.frejaeid.client.exceptions.FrejaEidClientPollingException;
+import java.time.Duration;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.*;
+
+/**
+ *
+ * @author User
+ */
+public class SignServiceTest {
+    
+    private static SignService instance;
+    
+    public SignServiceTest() {
+    }
+    
+    @BeforeAll
+    public static void setUpClass() {
+        System.out.println("Class setup");
+        try {
+            instance = new SignService();
+        } catch (FrejaEidClientInternalException ex) {
+            Logger.getLogger(SignServiceTest.class.getName()).log(Level.SEVERE, null, ex);
+            fail("Bad init");
+        }
+        System.out.println("Done");
+    }
+    
+    @AfterAll
```

Comment:
Nice to know how this annotations work, but I think we only need before all for now, you can remove rest of them.